filetests/isa/x86/run-const.clif

```diff
@@ -0,0 +1,11 @@
+test run
+
+function %test_compare_i32() -> b1 system_v {
```

Comment:
Is the Rust calling convention also system_v on windows machines? Maybe we should filter out tests that specify a calling convention different from the native one? (could be a follow-up)

Reply:
> Is the Rust calling convention also system_v on windows environments?

It is based on the native C abi I believe.

Reply:
Right, so this is Windows fastcall. (We can take care of this in a follow-up issue, but I'd prefer it to be fixed quickly so there are no strange test failures for Windows users)

Reply:
@bnjbvr and I talked about this some more and added the following check to avoid user error in specifying the calling convention: https://github.com/CraneStation/cranelift/pull/890/files#diff-f08bdadf8ad754c1568e3caa1d27a2f8R50-R57. Unfortunately we couldn't see an easy way to tell `parse_test` that we want a default other than `CallConv::Fast`, hence the second half of that expression. (I see it set here: https://github.com/CraneStation/cranelift/blob/master/cranelift-reader/src/parser.rs#L1004).